index.js

```diff
@@ -25,21 +23,14 @@ const { PORT, NODE_ENV, GRAVITY_API_URL, GRAVITY_ID, GRAVITY_SECRET } = process.
 
 const app = express()
 const port = PORT || 3000
-const sess = {
-  secret: GRAVITY_SECRET,
-  cookie: {},
-}
```

Comment:
Was MP using its API secret (w/Gravity) to encrypt its own session? I don't think that's necessary or advised. 😬

Reply:
Looks like it, yeah, I _think_ we can remove that client app. 

Not sure whether it's used anywhere in logging though.